[Workers] platform workers sites revision (#2807)
cloudflare/cloudflare-docs: products/workers/src/content/platform/sites/start-from-worker.md

  $ npm i @cloudflare/kv-asset-handler
  ```

> 3. Import the package’s code into your Worker script, and use it in the handler you would like to respond with static assets:

Reviewer: isn't a preposition missing in this sentence?

> 3. Import the package’s code into your Worker script, and use it in the handler you would like to respond with static assets:

Reviewer 2: ```suggestion
3. Import the package’s code into your Worker script and invoke it within your function handler to respond with static assets:
```
